### PR TITLE
Lab1

### PR DESCRIPTION
**Pull request by lexie98**

### CODE REVIEW

**Comment by lexie98** [comment]
@oliviatessa  Alexandrea (Lexie) Peterson's first lab

**Comment by ksl65** [comment]
I think your lab looks good. I like how you included the sigma values in a table, which makes it easy to understand how the sigma increases as the chosen measurement value increases. Also, the charts of binomial outcomes was very helpful in illustrating how the number of successes increased as the probability parameter increased.

File: Lab1Final.html [hunk 1]
```diff
@@ -0,0 +1,14066 @@
...
+fieldset[disabled] .btn-info.focus {
+  background-color: #5bc0de;
+  border-color: #46b8da;
+}
+.btn-info .badge {
+  color: #5bc0de;
+  background-color: #fff;
+}
+.btn-warning {
+  co
```
Comment (oliviatessa) [inline Lab1Final.html:13262]: Explain what this table means. Directly compare it to the z-table you copied from Wikipedia. Pretend I'm someone who has never learned this before and explain how z_table_test confirms what you expected.

Comment (oliviatessa) [inline Lab1Final.html:13364]: Explain why did you make this plot of cdf v sigma value. How is it helpful to your understanding of the problem?

Comment (oliviatessa) [inline Lab1Final.html:13462]: This doesn't explain why it's negative. And yes, you get the same probability for the standard normal distribution, but what about a different normal distribution?

Comment (oliviatessa) [inline Lab1Final.html:13609]: What does the value of 5.02 represent? (A: minimum value to be considered a signal) It's important to clearly state _why_ you are looking for the probability of getting a value greater than 5.02.

Comment (oliviatessa) [inline Lab1Final.html:13518]: What parameters does the lognormal distribution depend on?

Comment (oliviatessa) [inline Lab1Final.html:13671]: This is not how you convert from a probability to a sigma.

**Comment by oliviatessa** [comment]
Overall, I think you are still a little fuzzy on the meaning of sigma, specifically in relation to nonGaussian distributions. 

There are quite a few places throughout your writeup that could use more explanation.

**Pull request merged**